impl/src/main/java/com/sun/faces/cdi/CdiProducer.java

```diff
@@ -49,7 +49,7 @@
 
     private String id = this.getClass().getName();
     private String name;
-    private Class<?> beanClass = Object.class;
+	private Class<?> beanClass = Object.class;
```

Comment:
Tab needs to be spaces. Also in rest of PR.

Reply:
There were only 2. Nevertheless, sorry for those remainings.